com.sublimehq.SublimeText.appdata.xml

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+  <id>com.sublimehq.SublimeText</id>
+  <name>Sublime Text</name>
+  <project_license>LicenseRef-proprietary=https://www.sublimetext.com/eula</project_license>
+  <developer_name>Sublime HQ Pty Ltd</developer_name>
+  <summary>Sophisticated text editor for code, markup and prose</summary>
+  <metadata_license>CC0-1.0</metadata_license>
+  <url type="homepage">https://www.sublimetext.com/</url>
+  <url type="bugtracker">https://github.com/sublimehq/sublime_text/issues</url>
+  <url type="help">https://forum.sublimetext.com/</url>
+  <launchable type="desktop-id">com.sublimehq.SublimeText.desktop</launchable>
+  <description>
+    <strong>THIS IS A 3RD PARTY REDISTRIBUTION OF SUBLIME TEXT</strong>
```

Comment:
This tag is not supported. Please change the entire disclaimer to something like [Discord does](https://flathub.org/apps/details/com.discordapp.Discord). The description itself should be also expanded.